sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -425,6 +425,19 @@ public void putFloats(int rowId, int count, byte[] src, int srcIndex) {
     }
   }
 
+  @Override
+  public void putFloatsLittleEndian(int rowId, int count, byte[] src, int srcIndex) {
+    if (!bigEndianPlatform) {
```

Comment:
Really doesn't matter, but you could flip the comparison here to avoid the negation

Reply:
agreed, it definitely will look better.  will change in next commit.  thanks!

Reply:
Actually now I look at the existing `putDoublesLittleEndian()` and `putFloatsLittleEndian()` methods and they do `if (!bigEndianPlatform)` as well (and thus how I copied and pasted from those).

@srowen Do you think we can leave the negation alone for uniformity, or go ahead and change?

Reply:
Just leave it as-is for consistency, yeah. OK I see we have lots of "as little endian" variants so maybe this is the best approach for symmetry.

I dont' really know enough about this part to validate the change but it seems plausible.